x-pack/test/security_solution_cypress/cypress/e2e/explore/navigation/search_bar.cy.ts

```diff
@@ -22,13 +22,17 @@ import { getHostIpFilter } from '../../../objects/filter';
 
 import { hostsUrl } from '../../../urls/navigation';
 import { waitForAllHostsToBeLoaded } from '../../../tasks/hosts/all_hosts';
+import { waitForPageToBeLoaded } from '../../../tasks/common';
+import { LOADING_INDICATOR } from '../../../screens/security_header';
 
 // Failing: See https://github.com/elastic/kibana/issues/182932
-describe.skip('SearchBar', { tags: ['@ess', '@serverless'] }, () => {
+describe('SearchBar', { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(() => {
     login();
     visitWithTimeRange(hostsUrl('allHosts'));
     waitForAllHostsToBeLoaded();
+    waitForPageToBeLoaded();
+    cy.get(LOADING_INDICATOR).should('not.exist');
```

Comment:
The `waitForPageToBeLoaded()` already does `cy.getByTestSubj('globalLoadingIndicator').should('not.exist');` internally, is it necessary to do it again here?